website/settings/defaults.py

```diff
@@ -330,3 +330,5 @@ def parent_dir(path):
 assert (DRAFT_REGISTRATION_APPROVAL_PERIOD > EMBARGO_END_DATE_MIN), 'The draft registration approval period should be more than the minimum embargo end date.'
 
 PREREG_ADMIN_TAG = "prereg_admin"
+
+INSTITUTION_HIDE_UI_FLAG = True
```

Comment:
How about `ENABLE_INSTITUTIONS` for consistency with our other feature flags. Also, disabling this should disable the appropriate API endpoint(s), correct (not only hiding the UI)?